boards/BMS/Src/Io/Io_LTC6813.c

```diff
@@ -0,0 +1,374 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "Io_LTC6813.h"
+#include "Io_SharedSpi.h"
+#include "configs/Io_LTC6813Configs.h"
+
+enum CellVoltageRegisterGroups
+{
+    CELL_VOLTAGE_REGISTER_GROUP_A = 0,
```

Comment:
I think this would be `0` anyhow. Any particular reason we specified a value for only the first entry in the enum?

Reply:
Oh I think I based this off of Ken's code somewhere. Thought I was following some sort of convention for enums. I'll get rid of it.

Reply:
This should be fixed now